style.css

```diff
@@ -84,24 +97,21 @@
   }
 }
 
-.section2-content{
-  height: 60.4rem;
-   width: 32.7rem; 
-  background-image: url("./assets/header-mobile-view.png");
-  background-repeat: no-repeat;
-  background-size: cover;
+.section2-content {
+  margin: 7rem;
```

Comment:
- [ ] As the previous reviewer already stated, please fix the horizontal spacing in this section. You can reduce the <code>margin</code>
If you want to have different spacing on the vertical and the horizontal then you can use the shorthand<code>margin: margin_top, margin_right, margin_bottom, margin_left;</code>
You just substitute with the value you want 😇😇😇